### DISCUSSION TITLE
Backend Request: ClickHouse

### DISCUSSION THREAD

**Discussion by tomtom215** [moj-analytical-services/splink#2014] [Ideas]
Hello,

I would like to request https://github.com/ClickHouse/ClickHouse as an available backend database for similar reasons that DuckDB is so useful. 

Thank you!

**Comment by ADBond** [comment]
Hi @tomtom215 - realise this is some way down the road, but there is now a [separate package providing a Clickhouse interface to Splink](https://github.com/ADBond/splinkclickhouse), which you can install to use Clickhouse as the backend for Splink calculations. It is also available [on PyPI](https://pypi.org/project/splinkclickhouse/).

It probably won't be receiving the same level of support as Splink itself, but I will try to keep it in working order and respond to any bug reports.
